src/components/TheNavigation.vue

```diff
@@ -43,6 +43,16 @@
           State Channels
         </app-link>
       </li>
+      <li class="navigation__item">
+        <coming-soon-tooltip>
+          <app-link
+            class="navigation__link navigation__link--disabled"
+            target="_self"
+            to="#">
+            NFTs
+          </app-link>
+        </coming-soon-tooltip>
+      </li>
```

Comment:
```suggestion
```

NFTs should not be in the navigation bar, after the recent change.

Reply:
True, removed